src/__internal__/validation-message/validation-message.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { mount } from "enzyme";
+import ValidationMessage from ".";
+
+const render = (props) => mount(<ValidationMessage {...props} />);
+
+describe("ValidationMessage component", () => {
+  let wrapper;
+
```

Comment:
We need a test for when no props are passed, and tests for if `error`, `warning` and `info` aren't strings